templates/Template k8s-galera.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<zabbix_export>
+    <version>6.4</version>
+    <template_groups>
+        <template_group>
+            <uuid>a571c0d144b14fd4a87a9d9b2aa9fcd6</uuid>
+            <name>Templates/Applications</name>
+        </template_group>
+    </template_groups>
+    <templates>
+        <template>
+            <uuid>dfb8263cf05842f5bc3d8d7f3f11bcbf</uuid>
+            <template>Template k8s-galera mariadb status</template>
+            <name>Template k8s-galera mariadb status</name>
+            <groups>
+                <group>
+                    <name>Templates/Applications</name>
+                </group>
+            </groups>
+            <discovery_rules>
+                <discovery_rule>
+                    <uuid>b1134c1839a5481eb653131795ad65ce</uuid>
+                    <name>Galera cluster discovery</name>
+                    <type>ZABBIX_ACTIVE</type>
+                    <key>k8s-galera.discovery-clusters</key>
+                    <delay>1h</delay>
+                    <description>discovery of the mariadb clusters</description>
+                    <item_prototypes>
+                        <item_prototype>
+                            <uuid>0c03b14cfb904f8aaef6a74b493b3598</uuid>
+                            <name>galera cluster status {#CLUSTER} current_primary_pod_index</name>
+                            <type>DEPENDENT</type>
+                            <key>k8s-galera.current_primary_pod_index[{#CONTEXT},{#NAMESPACE},{#CLUSTER}]</key>
+                            <delay>0</delay>
+                            <history>1d</history>
+                            <value_type>TEXT</value_type>
+                            <trends>0</trends>
+                            <preprocessing>
+                                <step>
+                                    <type>JSONPATH</type>
+                                    <parameters>
+                                        <parameter>$.current_primary_pod_index</parameter>
+                                    </parameters>
+                                </step>
+                            </preprocessing>
+                            <master_item>
+                                <key>k8s-galera.values-clusters[{#CONTEXT},{#NAMESPACE},{#CLUSTER}]</key>
+                            </master_item>
+                            <trigger_prototypes>
+                                <trigger_prototype>
+                                    <uuid>3dc862e0a0ee4cd7ac9160a97e55fd34</uuid>
+                                    <expression>change(/Galera/k8s-galera.current_primary_pod_index[{#CONTEXT},{#NAMESPACE},{#CLUSTER}])=1</expression>
+                                    <name>Galera {#CONTEXT}:{#CLUSTER} primary pod changed</name>
+                                    <priority>DISASTER</priority>
+                                    <description>The primary Galera pod changed</description>
```

Comment:
Where did the uppercase `G` in galera suddenly come from?

Reply:
I'm assuming you're talking about the description, probably just went automatically with my also capitalizing it in the name (even though that was for a separate reason)